ROADMAP.md

```diff
@@ -32,27 +32,23 @@ Starting from scratch, create standard configuration that can be used with almos
 * Environment configuration
   - `.editorconfig`
   - `.gitignore`
-  - **Status**: Complete
 * Javascript configuration
   - `.eslintrc.js`
   - `.eslintignore`
-  - **Status**: Complete
 * Stylesheet configuration
   - Sass architecture specifications (e.g. 7+1, @use vs. @import)
   - `.stylelintrc.js`
-  - **Status**: Complete
 * Lint scripts
   - `eslint`
   - `stylelint`
-  - **Status**: Complete
 * Documentation
   - `README.md`
     - Standard data formats
   - `CHANGELOG.md`
   - Versioning strategy
   - Contribution instructions
-  - **Status**: Complete
 
+These changes were implemented in https://github.com/quicksolutions/starter-kit/pull/1.
 
 ## Phase 1: Pre-rendered Static Websites
```

Comment:
In keeping with the Phase 0 suggested change, this and all other phase headings should have their own status labels.